extensions/SimonShiki/scopeVar.js

```diff
@@ -0,0 +1,152 @@
+// Name: Scope Variables
+// ID: scopeVar
+// Description: Manage your data inside your stack (or substack).
+// By: SimonShiki <https://scratch.mit.edu/users/SinanGentoo/>
+// Original: Skyhigh173
+
+(function (Scratch) {
+  "use strict";
+  if (!Scratch.extensions.unsandboxed) {
+    throw new Error(`
+            "Scope Variables" must be loaded in a unsandboxed environment.
+        `);
+  }
+  const { BlockType, ArgumentType, Cast } = Scratch;
+
+  class ScopeVar {
+    getInfo() {
+      return {
+        id: "scopeVar",
+        name: "Scope Variables",
+        collaborator: "Skyhigh173, SimonShiki",
```

Comment:
Not exactly sure where you got "collaborator" and "description" for, but those are already handled in the metadata comments above.

Reply:
This field will be handled by other scratch mods, so I kept it. If this affects future compatibility on TurboWarp I will remove it.